quickwit/quickwit-metastore/src/metastore/file_backed_metastore/mod.rs

```diff
@@ -994,9 +1005,13 @@ mod tests {
     use super::*;
     use crate::metastore::MetastoreServiceStreamSplitsExt;
     use crate::tests::DefaultForTest;
-    use crate::{metastore_test_suite, IndexMetadata, ListSplitsQuery, SplitMetadata, SplitState};
+    use crate::{
+        file_backed_index_shards_test_suite, metastore_test_suite, IndexMetadata, ListSplitsQuery,
+        SplitMetadata, SplitState,
+    };
 
     metastore_test_suite!(crate::FileBackedMetastore);
+    file_backed_index_shards_test_suite!(crate::FileBackedMetastore);
```

Comment:
What are you trying to achieve here? There's already a `metastore_test_suite!` that runs for both implementations and covers the shard API. It is also possible to turn on/off shard tests independantly (see `quickwit-metastore/src/tests/shard.rs`)